tjpcov/nmt_tools.py

```diff
@@ -0,0 +1,438 @@
+#!/usr/bin/python
+import healpy as hp
+import os
+import pymaster as nmt
+import numpy as np
+
+
+
+
+def get_tracer_dof(sacc_data, tracer):
```

Comment:
I can see where dof is being used. However, I do not think dof is standard terminology and I believe it is not needed. We can simply use the sum of spins, 0+0== g X g or kappa X kappa, 0+2== g or kappa X Shear,  2+2 == shear-shear. 

I will prefer not to have an extra quantity defined. If you think it is necessary to talk to namaster, I will recommend adding a brief explanation here.